src/components/ResultDisplay.tsx

```diff
@@ -69,3 +72,14 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 function counted(count: number, text: string) {
 	return `${count} ${text}${count === 1 ? "" : "s"}`;
 }
+
+function summed(result: EstimatedPackage[]) {
+	let total = 0;
+	result.forEach((num) => {
+		total += num.estimatedMoney;
+	});
```

Comment:
[Suggestion] Not a blocker IMO, but if you want to be all snazzy...

```ts
const total = result.reduce((total, current) => total + current.estimatedMoney, 0);
```

Reply:
All about the snazz😎  Yes, I agree reduce is a more appropriate method here

Reply:
We could also remove a couple more lines and do:
```
function sumEstimateFunding(result: EstimatedPackage[]) {
	const total = result
		.reduce((total, current) => total + current.estimatedMoney, 0)
		.toLocaleString("en-US", {
			maximumFractionDigits: 0,
		});
```

Reply:
<img alt="Jack Nicholson slowly nodding his head yes with a grin" src="https://media3.giphy.com/media/KDVswimTNahWzcd7sV/giphy.gif"/>